sixMans/sixMans.py

```diff
@@ -221,6 +221,29 @@ async def q(self, ctx):
         if six_mans_queue._queue_full():
             await self._randomize_teams(ctx, six_mans_queue)
 
+    @commands.guild_only()
+    @commands.command(aliases=["forcequeue"])
+    @checks.admin_or_permissions(manage_guild=True)
+    async def fq(self, ctx, *, member: discord.Member):
```

Comment:
I'm not sure if you noticed the queueAll command above, but it seems like this new command tests the same functionality that the queueAll command is there to test so I don't think this is necessary unless there's something new I'm missing.

Reply:
I did see it. However, I didn't want members of my test server getting pinged for my tests. So I manually queued specific bots or people who didn't mind.

Happy to have it removed, but it doesn't make a huge difference.

Reply:
The queueAll command takes in a list as a parameter so you can specify the list of members you want added. I'm not sure if there was confusion on that or not. The only difference I see between this command and the queueAll command is that queueAll takes in at least one member and potentially multiple whereas this one takes in only one at a time. I think it's redundant in that queueAll can be used in the same manner if desired.

Reply:
Unless there's some functionality here that I'm missing I think this should be removed and then an alias for queueAll can be added to work with the same command name.

Reply:
In that case, there's no difference and it should be removed. I misunderstood the behavior of queueAll. I'll have the forcequeue command removed.